actions/check-product-changes/index.js

```diff
@@ -41,7 +43,11 @@ async function main(params) {
     // this might not be updated and action execution could be permanently skipped
     // a ttl == function timeout is a mitigation for this risk
     await stateMgr.put('running', 'true', { ttl: 3600 });
-    activationResult = await poll(params, { stateLib: stateMgr, filesLib }, logger);
+    activationResult = await poll(
+        { ...params, __cfg: cfg },
```

Comment:
does the `poller.js` use the `__cfg`? I cannot see this change, did you commit it?